github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -30,7 +30,7 @@ periodics:
           - name: BASE_REPOID
             value: devel_kubic_libcontainers_stable
           - name: CRIO_VERSIONS
-            value: "1.20,1.21,1.22,1.23"
+            value: "1.20:1.20.6,1.20,1.21:1.21.5,1.21,1.22:1.22.2,1.22,1.23:1.23.1,1.23"
```

Comment:
I am not familiar with the syntax here, but the last `,1.23` looks superfluous to me. Is it?

Reply:
the syntax is from the cri-o installation docs - https://github.com/cri-o/cri-o/blob/0ba47c9b3e52eee95d898be84500c38c9fe032c9/install.md?plain=1#L63 

The last 1.23 will take the latest cri-o release which would be 1.23.2.

Reply:
Nice :+1:

Reply:
I have tested these values with the curl command from mirror-crio.sh and they complete successfully.

https://github.com/kubevirt/project-infra/blob/7009bef45c1fb9f7a5c0a6c1a453943657058971/hack/mirror-crio.sh#L20